src/main/resources/templates/languages/cy/common/nonStrategicCommon.json

```diff
@@ -6,9 +6,39 @@
   "weeklyListDate": "Rhestr ar gyfer yr wythnos yn dechrau ar ",
   "lastUpdated": "Diweddarwyd ddiwethaf ",
   "at": " am ",
+  "venueNameRolls": "Adeilad Rolls",
+  "addressLineRolls1": "Fetter Lane, London",
+  "addressLineRolls2": "EC4A 1NL",
+  "venueNameRcj": "Llysoedd Barn Brenhinol",
+  "addressLineRcj1": "Strand, London",
+  "addressLineRcj2": "WC2A 2LL",
   "importantInformationHeading": "Gwybodaeth bwysig",
+  "remoteHearingsHighCourtJudgeHeading": "Gwrandawiadau o bell gerbron Barnwr yr Uchel Lys",
+  "remoteHearingsHighCourtJudgeText": "Os yw cynrychiolydd o'r cyfryngau neu aelod o'r cyhoedd yn dymuno mynychu'r gwrandawiad, dylent gysylltu â'r swyddfa restru yn chanceryjudgeslisting@justice.gov.uk a fydd yn eu rhoi mewn cysylltiad â'r unigolyn perthnasol.",
+  "remoteHearingsChanceryMasterHeading": "Gwrandawiadau o bell gerbron Meistr y Siawnsri",
+  "remoteHearingsChanceryMasterText": "Os yw cynrychiolydd o'r cyfryngau neu aelod o'r cyhoedd yn dymuno mynychu'r gwrandawiad o bell, dylent gysylltu â chancery.mastersappointments@justice.gov.uk a fydd yn eu rhoi mewn cysylltiad â'r unigolyn perthnasol.",
+  "remoteJudgementsHeading": "Dyfarniadau o bell",
+  "remoteJudgementsText": "Traddodi o Bell: Bydd y dyfarniad hwn yn cael ei draddodi o bell trwy gylchrediad i'r partïon neu eu cynrychiolwyr trwy e-bost a'i ryddhau i'r Archifau Cenedlaethol. Dylai copi o'r dyfarniad ar ffurf derfynol fel y'i rhoddwyd fod ar gael ar wefan yr Archifau Cenedlaethol yn fuan wedyn. Gall aelodau'r cyfryngau gael copi ar gais trwy e-bost i'r Swyddfa Farnwrol press.enquiries@judiciary.uk.",
   "observeHearingLinkText": "Arsylwi gwrandawiad llys neu dribiwnlys fel newyddiadurwr, ymchwilydd neu aelod o'r cyhoedd",
   "observeHearingLink": "https://www.gov.uk/guidance/observe-a-court-or-tribunal-hearing",
+  "tableHeadersCommon1": [
```

Comment:
I think tableHeadersCommon1 and tableHeadersCommon2 would be good to be renamed to something to describe the lists they are for. Same comment for the english file